templates/dev.rake

```diff
@@ -1,4 +1,4 @@
-if Rails.env.development? || Rails.env.test?
+unless Rails.env.production?
```

Comment:
I just personally don't like seeing "production" anywhere it's NOT production. From an Ops perspective, I've been burned by that before.

That said, I'd suggest renaming "dev:prime" to "db:prime" and an env flag of `PRIMEABLE_ENV=true` -- but I'll leave it up to you to decide.